examples/train.py

```diff
@@ -173,6 +174,15 @@ def train(args):
 
 
 if __name__ == "__main__":
+    if sys.version_info < (3,7) or sys.version_info >= (3,8):
+        os.system("")
```

Comment:
The os.system("") isn't needed, right?

Reply:
It was recommended on a SO post to ensure compatibility with Windows. I don't have a window computer at hand to test it (see [here](https://stackoverflow.com/questions/12492810/python-how-can-i-make-the-ansi-escape-codes-to-work-also-in-windows)).

Reply:
Thanks, that's not what I expected!